nearuplib/nodelib.py

```diff
@@ -18,27 +20,75 @@
     download_genesis,
     latest_genesis_md5sum,
     new_release_ready,
+    prompt_flag,
 )
 from nearuplib.watcher import is_watcher_running, run_watcher, stop_watcher
 from nearuplib.tailer import next_logname
 
 
-def init_near(home_dir, binary_path, chain_id, init_flags):
+def read_validator_key(home_dir):
+    key_path = os.path.join(home_dir, 'validator_key.json')
+    if not os.path.exists(key_path):
+        return None
+
+    with open(key_path) as key_file:
+        return json.loads(key_file.read())
```

Comment:
`json.load(key_file)`

PS. Another way to write this function would be:

    key_path = os.path.join(home_dir, 'validator_key.json')
    try:
        with open(key_path) as key_file:
            return json.load(key_file)
    except FileNotFoundError:
        return None

I kinda prefer this form because it avoids time-of-check/time-of-use though it doesn’t really matter in this case.

Reply:
yeah I like that better, too